recipes/calceph/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX --enable-thread=yes --enable-fortran=no --enable-shared=yes --enable-static=no
+
+make -j${CPU_COUNT} VERBOSE=1
+
+make install
```

Comment:
```suggestion
make install

rm -fr $PREFIX/libexec
rm -fr $PREFIX/share/doc/calceph/html
```
MEX files are probably not needed because you can't install MATLAB into a conda-environment?

Is is necessary to ship 75MB of documentation source files?  $PREFIX/share/doc/calceph already contains the compiled docs as PDF for only 2 MB.